randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2122,31 +2143,37 @@ Extra - asset_id: collision_camera_019
   * Layers: default
   * Open Passage to Early Gravity Speedboost Room 1/Dock to Early Gravity Speedboost Room 2 (Lower)
   > Slope Top
-      Speed Booster or Walljump (Beginner) or Simple IBJ or Use Spin Boost
+      After Burenia - Early Gravity Speed Blocks Destroyed
 
 > Water Pit; Heals? False
   * Layers: default
   > Dock to Early Gravity Speedboost Room 1 (Upper)
       Any of the following:
           Gravity Suit and Simple IBJ
-          All of the following:
-              Spider Magnet
-              Water Bomb Jump (Beginner) or Use Spin Boost
+          Spider Magnet and Perform WBJ
   > Slope Top
       Use Spin Boost
 
 > Slope Top; Heals? False
   * Layers: default
   > Dock to Early Gravity Speedboost Room 1 (Upper)
       Any of the following:
-          Space Jump
+          Flash Shift or Use Spin Boost
```

Comment:
Should Flash Shift require Beginner Movement/Wall Jump here (not sure which to put)? You have to do a quick respin to make it up after the flash